Makefile

```diff
@@ -71,6 +71,8 @@ else
 	scBuildImageTarget = .scBuildImage
 endif
 
+NON_VENDOR_DIRS = $(shell $(DOCKER_CMD) glide nv)
```

Comment:
I forgot to check if this has to be defined after the `NO_DOCKER` `DOCKER_CMD` definition, but I figure it is clearer if it follows. Using `NO_DOCKER` still works.